content/learning/operator/operator.md

```diff
@@ -65,10 +65,100 @@ Kubebuilder will generate the controller code only after generating CRDs, so eve
 The kubebuilder framework uses the controller-runtime library to communicate with the Kubernetes API. The documentation of the library is sparse across different packages, here are some useful ones:
 1. [controller-runtime](https://godoc.org/sigs.k8s.io/controller-runtime "controller-runtime") general godoc page
 2. controller-runtime\'s [client](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client "client") used to interact with the k8s API
-3. the [buider](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/builder "buider") used in the `SetupWithManager` function to select which resources to watch
+3. the [builder](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/builder "builder") used in the `SetupWithManager` function to select which resources to watch
 4. the [request](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/reconcile#Request "request") received in the reconciler function
 5. k8s [corev1](https://godoc.org/k8s.io/api/core/v1 "corev1") in case you need to interact with core k8s kind (pods, deployments, configmaps etc)
 
+## Useful tips
+
+Here are some useful suggestions if you are new to programming an operator:
+
+1. You may need to watch more than one resource, e.g., when you have to react to different conditions.
+This leads the operator to wake up when either one of the watched resources is modified.
+A possible example is the following: your controller manages a CRD "laboratory" in which a label identifies the status (running or stopped) of a VirtualMachine (VM), which can be started or stopped directly by the user.
```

Comment:
In this example it seems that we're watching for a single resource. Can you clarify why this example applies to the case under consideration here, i.e., "You may need to watch more than one resource"?

Reply:
This example has been updated. Thank you for your suggestion!